.github/workflows/test.yml

```diff
@@ -109,12 +109,14 @@ jobs:
         echo ::set-output name=vault-token::$VAULT_TOKEN
         echo ::add-mask::$VAULT_TOKEN
 
-    - name: Render config
+    - name: Write config
       if: matrix.gradleTask != 'unitTest'
-      uses: ./.github/actions/render-config
+      id: config
+      uses: ./.github/actions/write-config
       with:
         test-env: ${{ steps.set-env-step.outputs.config-env }}
```

Comment:
Also remove `test-env` here.

FWIW - It doesn't actually seem to break this workflow. Message in GHA logs is `Warning: Unexpected input(s) 'test-env', valid inputs are ['vault-token', 'target']`